collections/misc/collprofiles.php

```diff
@@ -389,6 +389,9 @@ function directSubmitAction(e) {
 										<a href="#" onclick="$('li.traitItem').show(); return false;">
 											<?= $LANG['TRAIT_CODING_TOOLS'] ?>
 										</a>
+										<a href="#" onclick="$('li.traitItem').show(); return false;"> 
+											<img class="seemore" src="../../images/tochild.png" style="width:13px;height:13px;">
```

Comment:
We're trying to avoid inline styling when we can. There's already a <style> element in collections/misc/collprofiles.php. Since this element isn't being used on other pages yet, new classes can be added to the <style> element directly in collections/misc/collprofiles.php (e.g., the way `.quicksearch-container {` is defined).